### PR TITLE
add nodeupgradejob related rbac to cloudcore deployment

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
add nodeupgradejob related rbac to cloudcore deployment
follow up https://github.com/kubeedge/kubeedge/pull/4004
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: build/cloud/03-clusterrole.yaml [hunk 1]
```diff
@@ -36,3 +36,6 @@ rules:
 - apiGroups: ["networking.istio.io"]
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["operations.kubeedge.io"]
+  resources: ["nodeupgradejobs", "nodeupgradejobs/status"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```
Comment (fisherxu) [inline build/cloud/03-clusterrole.yaml:41]: `delete`  should not be used?

Comment (gy95) [reply build/cloud/03-clusterrole.yaml:41]: yes

Comment (fisherxu) [inline build/cloud/03-clusterrole.yaml:41]:
```suggestion
  verbs: ["get", "list", "watch",  "update", "patch"]
```

Comment (gy95) [reply build/cloud/03-clusterrole.yaml]: fixed

**Comment by gy95** [comment]
cc @fisherxu PTAL

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Comment by fisherxu** [comment]
Let's cherry-pick it to release-1.12

**Pull request merged**